package.bzl

```diff
@@ -35,11 +35,11 @@ def rules_nodejs_dev_dependencies():
     """
 
     # Dependencies for generating documentation
+    # TODO(manekinekko): switch to https://github.com/bazelbuild/rules_sass/ when the changes have been released
```

Comment:
changes -> should point to a PR /commit so we know how to go check if it's released yet

Reply:
👍